### PR TITLE
(SOFTWARE-3058) Add troubleshooting YUM repos

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/services/troubleshoot-yum-repos.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+6.  If **all** timestamps are more than 6 hours old, something may be
+    wrong with cron or its mash entries:
+    1.  If possible, verify that cron is running and that the cron
+        entries for mash are still present; if not, try to restore
+        things
+    2.  Otherwise, create a Freshdesk ticket with a subject like "Repo update
+        logs are too old on repo.opensciencegrid.org" and with relevant details in the
+        body
+    3.  Assign the ticket to the "Software" group
+    4.  Email Carl Edquist <edquist@cs.wisc.edu> and CC Tim
```
Comment (brianhlin) [inline docs/services/troubleshoot-yum-repos.md]: Let's get rid of this step

File: docs/services/troubleshoot-yum-repos.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+1.  Log onto `repo.opensciencegrid.org` to view logs from `mash` updates
+2.  Examine the "Last modified" timestamp of all of the
+    `/var/log/repo/update_repo.*.log` files
+3.  If the timestamps are all less than 2 hours old, life is good and
+    you can skip the remaining steps below
+4.  Otherwise, examine the "Last modified" timestamp of the
+    `/var/log/repo/update_all_repos.err` file
+5.  If the `update_all_repos.err` timestamp is current, there may be a
+    `mash` process that is hung; see the Troubleshooting steps below
```
Comment (brianhlin) [inline docs/services/troubleshoot-yum-repos.md]:
```suggestion
    `mash` process that is hung; see [this section](#troubleshooting-and-mitigation]
```

File: docs/services/troubleshoot-yum-repos.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    Then verify that the old processes are gone using the same `ps` command as above:
+
+        # ps -C mash -o pid,ppid,pgid,start,command
+          PID  PPID  PGID  STARTED COMMAND
+
+1.  If any part of this process does not look or work as expected:
+    1.  Create a Freshdesk ticket with a subject like "Problems with mash on
+        repo.opensciencegrid.org" and with relevant details in the body
+    2.  Assign the ticket to the "Software" group
+    3.  Email Carl Edquist <edquist@cs.wisc.edu> and CC Tim
```
Comment (brianhlin) [inline docs/services/troubleshoot-yum-repos.md]: Let's get rid of this step

**Comment by brianhlin** [comment]
Ping @timtheisen

**Comment by timtheisen** [comment]
Superseded put PR #193

**Pull request closed without merging**